src/libraries/System.Text.RegularExpressions/tests/RegexReductionTests.cs

```diff
@@ -7,7 +7,7 @@
 namespace System.Text.RegularExpressions.Tests
 {
     [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Many of these optimizations don't exist in .NET Framework.")]
-    [SkipOnPlatform(TestPlatforms.Browser, "Internal members accessed via reflection may have been trimmed away")]
+    [ConditionalClass(typeof(PlatformDetection), nameof(PlatformDetection.IsNotBuiltWithAggressiveTrimming))]
```

Comment:
why both here and below? not that it matters

Reply:
I don't fully know how xunit runs things / queries for attributes and whether it might cause static construction of a test class even if the tests are disabled via attribution.